Source/WebCore/platform/graphics/cocoa/MediaPlaybackTargetContext.h

```diff
@@ -57,21 +56,43 @@ class MediaPlaybackTargetContext {
     MediaPlaybackTargetContext() = default;
     WEBCORE_EXPORT explicit MediaPlaybackTargetContext(RetainPtr<AVOutputContext>&&);
 
-    MediaPlaybackTargetContext(RetainPtr<NSData>&& serializedOutputContext, bool hasActiveRoute)
-        : m_type(Type::SerializedAVOutputContext)
-        , m_serializedOutputContext(WTFMove(serializedOutputContext))
-        , m_cachedHasActiveRoute(hasActiveRoute)
-    {
-        ASSERT(m_serializedOutputContext);
-    }
-
     MediaPlaybackTargetContext(const String& mockDeviceName, MockState state)
         : m_type(Type::Mock)
         , m_mockDeviceName(mockDeviceName)
         , m_mockState(state)
     {
     }
 
+    struct NonPlatformData {
+        String mockDeviceName;
+        MockState mockState;
+    };
+
+    struct PlatformData {
+        RetainPtr<AVOutputContext> outputContext;
+    };
+
+    using PlatformDataVariant = std::variant<PlatformData, NonPlatformData>;
```

Comment:
PlatformDataVariant is not a good name IMO because it may also represent NON platform data. We usually use `IPCData` for this.